packages/utils/src/customer-requests/medical-records-local-with-brief.ts

```diff
@@ -0,0 +1,84 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { Bundle, Resource } from "@medplum/fhirtypes";
+import {
+  createMRSummaryBriefFileName,
+  createMRSummaryFileName,
+} from "@metriport/core/domain/medical-record-summary";
+import { bundleToBrief } from "@metriport/core/external/aws/lambda-logic/bundle-to-brief";
+import { bundleToHtml } from "@metriport/core/external/aws/lambda-logic/bundle-to-html";
+import { S3Utils } from "@metriport/core/external/aws/s3";
+import fs from "fs";
+
+const s3Client = new S3Utils("us-east-2");
+
+const bucketName = "medical-documents-staging";
+const cxId = "";
+const patientId = "";
+
+async function main() {
+  const bundle = fs.readFileSync("test-fhir-fry.json", "utf8");
+  const bundleParsed = JSON.parse(bundle);
+
+  const brief = await bundleToBrief(bundleParsed as Bundle<Resource>, cxId, patientId);
+
+  if (!cxId || !patientId) throw new Error("cxId or patientId is missing");
+  const briefFileName = createMRSummaryBriefFileName(cxId, patientId);
+  const htmlFileName = createMRSummaryFileName(cxId, patientId, "html");
+
+  // Response from FHIR Converter
+  const html = bundleToHtml(bundleParsed, brief);
+  await storeMrSummaryAndBriefInS3({
+    bucketName,
+    htmlFileName,
+    briefFileName,
+    html,
+    brief,
+  });
+
+  fs.writeFileSync("test.html", html);
+}
+
+main();
+
+async function storeMrSummaryAndBriefInS3({
```

Comment:
NABD now, but keeping this feature around we prob want to have the least code possible on the lambda and most of it on `core`. Then the utils can use it as well and we don't have to duplicate this.

If you prefer to leave as is for now, can we add a TODO on both functions to improve on this later, please?